src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -88,6 +89,12 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig, protected
     UUID.fromString(dop.getMetadata.get("clusterUuid").toString)
   }
 
+  def getEmailFromAccessToken(accessToken: String)(implicit executionContext: ExecutionContext): Future[String] = {
+    val oauth2 = new Builder(httpTransport, jsonFactory, null).setApplicationName(dataprocConfig.applicationName).build()
```

Comment:
Maybe maybe make a `private lazy value oauth2 = new Oauth2.Builder(...)` like we do with other services?